### PR TITLE
XS✔ ◾ Update rule.md

### PR DESCRIPTION
**Pull request by ae-cher-ssw**
adding screenshots

> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️

> 2. What was changed?

✏️

> 3. Did you do pair or mob programming?

✏️

### CODE REVIEW

**Comment by tiagov8** [comment]
@ae-cher-ssw 

Feel free to ping me on Teams if you need help with this edit

File: rules/connect-client-im/rule.md [hunk 1]
```diff
@@ -32,16 +32,24 @@ Once you’ve arranged an Initial Meeting or before a Spec Review, follow these
...
 4. Create the chat
 
+![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/147477898/538c67a7-d941-4a6b-a072-305169d498b2)
+
+**Figure:  Client with open tenant**
+
 ### If the client has an open tenant, you should send a welcome message in Teams.
 
-::: greybox  
+:::
```
Comment (sethdailyssw) [inline rules/connect-client-im/rule.md:41]: You need 'greybox' here to make it a box

```suggestion
:::  greybox
```

File: rules/connect-client-im/rule.md [hunk 2]
```diff
@@ -32,16 +32,24 @@ Once you’ve arranged an Initial Meeting or before a Spec Review, follow these
 3. Enter the client’s email address
 4. Create the chat
 
+![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/147477898/538c67a7-d941-4a6b-a072-305169d498b2)
```
Comment (sethdailyssw) [inline rules/connect-client-im/rule.md:35]:
```suggestion
![image](teams-rule.png)
```

File: rules/connect-client-im/rule.md [hunk 3]
```diff
@@ -32,16 +32,24 @@ Once you’ve arranged an Initial Meeting or before a Spec Review, follow these
...
+
+**Figure:  Client with open tenant**
```
Comment (sethdailyssw) [inline rules/connect-client-im/rule.md:37]:
```suggestion
```

**Review by sethdailyssw — APPROVED** [review]

**Pull request merged**